api/SpeechSynthesis.json

```diff
@@ -497,7 +448,10 @@
         "__compat": {
           "description": "<code>voiceschanged</code> event",
```

Comment:
The data for this entry doesn't match the removed `onvoiceschanged` entry. From source grep I can say that WebKit doesn't support the event, so that should be changed below, but the lack of support in Opera was probably just a mistake.

Reply:
Thanks! 👍